Source/DafnyDriver/Legacy/SynchronousCliCompilation.cs

```diff
@@ -439,7 +439,7 @@ public class SynchronousCliCompilation : IDisposable {
       }
 
       var result =
-        await DafnyMain.BoogieOnce(errorReporter, options, output, engine, baseName, moduleName, program, programId);
+        await await DafnyMain.LargeStackFactory.StartNew(() => DafnyMain.BoogieOnce(errorReporter, options, output, engine, baseName, moduleName, program, programId));
```

Comment:
I admit I just imitated what I saw elsewhere, but I didn't grok why we need the double `await`